implants/lib/eldritch/src/crypto/aes_encrypt_file_impl.rs

```diff
@@ -0,0 +1,67 @@
+use std::fs::File;
+use std::io::{BufReader, Read, Write};
+
+use anyhow::Result;
+use crypto::aes::{self, KeySize};
+use crypto::symmetriccipher::SynchronousStreamCipher;
+
+pub fn encrypt_file(src: String, dst: String, key: String) -> Result<()> {
+    let bytes = key.as_bytes();
+    let mut cipher = aes::ctr(KeySize::KeySize128, bytes, bytes);
+    let f = File::open(src)?;
+    let mut reader = BufReader::new(f);
+    let mut buffer: Vec<u8> = Vec::new();
+    reader.read_to_end(&mut buffer)?;
```

Comment:
We may want to switch to a streaming encryption for better performance with large files.
I've seen a few c2s crash trying to encrypt Medium to large files especially on low powered VMs in competitions.